manifests/base/crds/kubeflow.org_tfjobs.yaml

```diff
@@ -6916,6 +6916,53 @@ spec:
                       description: The number of pods which reached phase Failed.
                       format: int32
                       type: integer
+                    labelSelector:
```

Comment:
If other operators does not support scale subresource, then it is not needed. Do you mean we should support the subresource for other operators?

Reply:
After the update to kubeflow/common, every time we run `make generates` in this repo, it shall always add `labelSelector` to other APIs. I would suggest to ignore such changes from APIs except `PyTorchJob` in this pr.

Reply:
It is hard to ignore it since the code generator reads the API definition and generate the corresponding code automatically. And the field `labelSelector` is a pointer thus it is optional. It does not affect existing CRDs